### PR TITLE
Simplify IANA registration section.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- simplifies IANA field registration section
- obsolete Digest

### CODE REVIEW

File: draft-ietf-httpbis-digest-headers.md [hunk 1]
```diff
@@ -584,6 +584,23 @@ validation of the algorithm types, number of validations, or the size of content
 
 # IANA Considerations
 
+IANA is requested to update one registry and create one new registry.
```
Comment (LPardue) [inline draft-ietf-httpbis-digest-headers.md:587]:
```suggestion
## HTTP Field Name Registration
```

File: draft-ietf-httpbis-digest-headers.md [hunk 2]
```diff
@@ -584,6 +584,23 @@ validation of the algorithm types, number of validations, or the size of content
...
+
+Please update the
```
Comment (LPardue) [inline draft-ietf-httpbis-digest-headers.md:589]:
```suggestion
IANA is asked to update the
```

File: draft-ietf-httpbis-digest-headers.md [hunk 3]
```diff
@@ -584,6 +584,23 @@ validation of the algorithm types, number of validations, or the size of content
...
+({{SEMANTICS}}) according to the table below:
+
+|--------------------------------|-----------|----------------------------------------------------------------|
+| Field Name                     | Status    |                     Reference                                  |
+|--------------------------------|-----------|----------------------------------------------------------------|
+| Representation-Digest          | permanent | {{representation-digest}} of this document                     |
+| Content-Digest                 | permanent | {{content-digest}} of this document                            |
+| Want-Representation-Digest     | permanent | {{want-fields}} of this document                               |
+| Want-Content-Digest            | permanent | {{want-fields}} of this document                               |
+| Digest                         | obsoleted | [RFC3230], {{obsolete-3230}} of this document                  |
```
Comment (LPardue) [inline draft-ietf-httpbis-digest-headers.md:600]: Also need to obsolete `Want-Digest`.

**Review by LPardue — CHANGES_REQUESTED** [review]
Thanks for this, it's almost there. 

I suggest we just have two sections that ask IANA to do things directly, rather than giving any prelude to those instructions.

File: draft-ietf-httpbis-digest-headers.md (hunk 3 above)
Comment (ioggstream) [reply draft-ietf-httpbis-digest-headers.md:600]:
```suggestion
| Digest                         | obsoleted | [RFC3230], {{obsolete-3230}} of this document                  |
| Want-Digest                    | obsoleted | [RFC3230], {{obsolete-3230}} of this document                  |

```

File: draft-ietf-httpbis-digest-headers.md [hunk 4]
```diff
@@ -584,6 +584,25 @@ validation of the algorithm types, number of validations, or the size of content
...
+| Want-Digest                    | obsoleted | [RFC3230], {{obsolete-3230}} of this document                  |
+
```
Comment (ioggstream) [inline draft-ietf-httpbis-digest-headers.md:602]:
```suggestion
```

**Pull request merged**